spec/support/a_framework_assessment.rb

```diff
@@ -42,65 +41,65 @@
     end
 
     it 'returns error if move is not associated to a profile' do
-      create(:framework, version: '1.2.1')
+      create(:framework, name: assessment_type.to_s.dasherize, version: '1.2.1')
```

Comment:
I think it might read more nicely and be less repetitive if this were assigned to a `let` instead? Something like

```suggestion
      let(:framework_name) { assessment_type.to_s.dasherize }
      
      ...
      
      create(:framework, name: framework_name, version: '1.2.1')
```

Reply:
I would prefer keeping it defined in the context of each spec, mainly due to multiple variations of the framework being defined across each specs